helm/coredns-app/templates/deployment-masters.yaml

```diff
@@ -73,6 +73,8 @@ spec:
         securityContext:
           allowPrivilegeEscalation: false
           capabilities:
+            add:
+            - NET_BIND_SERVICE
```

Comment:
This is required as the image now sets this cap (https://github.com/coredns/coredns/pull/5969/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R13)

Reply:
Does this need some kyverno exception in regards to PSS? I faintly remember this isn't a problem in kube-system, right?
I also remember there was a mechanism for restricting the installation namespace of a chart/app. If above is true, the chart should have something like this in place (if not already)

Reply:
Actually we are using this as a default in Ingress NGINX Controller for a while now. 😅

Reply:
Indeed, as Marco mentioned, it is allowed in PSS, even under restricted. 

> Containers must drop ALL capabilities, and are only permitted to add back the NET_BIND_SERVICE capability. [This is Linux only policy](https://kubernetes.io/docs/concepts/security/pod-security-standards/#os-specific-policy-controls) in v1.25+ (.spec.os.name != "windows")

You can find more info here: https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted